web/templates/invite_user_modal.hbs

```diff
@@ -97,4 +97,19 @@
             </div>
         </div>
     {{/if}}
+    {{#if is_admin}}
```

Comment:
Should we show this if the organization setting is not checked? I am not sure if this has been discussed before.